cmd/yopass/main.go

```diff
@@ -37,10 +37,16 @@ Examples:
 Website: %s
 `
 
+var (
+	defaultAPI = "https://api.yopass.se"
+	defaultURL = "https://yopass.se"
+)
+
 func init() {
-	// Defaults
-	viper.SetDefault("api", "https://api.yopass.se")
-	viper.SetDefault("url", "https://yopass.se")
+    // Use build-time values if set; otherwise, fall back to hardcoded defaults. 
```

Comment:
could you make sure this comment is correctly indented :)

Reply:
It should be fixed now :)